### ISSUE TITLE
Unable to click on the stash tab overlay [FIXED]

### ISSUE THREAD

**Issue by jamesgaunt** [ChaosRecipeEnhancer/ChaosRecipeEnhancer#51]
The stash tab overlay doesn't seem to detect any clicks on it. I can't click the edit button, and I when clicking on an highlighted item with the XXX indicator it doesn't register the item has been clicked.

Any tips on how to get this working?

**Comment by jamesgaunt** [comment]
Fixed this. It was because of multiple monitors and my main display was on the right. But I play POE on my middle monitor and so the overlay X offset was negative. For some reason this broke things.

Making my middle monitor the main display, and manually clearing the various offsets held in the user.config file got things working again.

Great tool BTW.

**Comment by jjepaah** [comment]
Hi, I'm having the same issue and I'm using 2 monitors (right one's the main). Where did you find the user.config file and how did you figure out the right offsets?
Thanks!

**Comment by jamesgaunt** [comment]
Hi. User config was in C:\Users\<username>\AppData\Local\EnhancePoE\<gibberish name>\1.0.0.0\user.config

I just deleted the sections that mentioned offsets (there are 2 or 3 of them through the file) and restarted and it reset to default - i.e. on main monitor. You can probably just delete the entire user config if you don't care about having to reset your other settings.

**Comment by kosace** [comment]
did you have a previous version installed? or did you change the main monitors while it being installed?

**Comment by jamesgaunt** [comment]
I installed 1.13 which I think is the latest. I didn't change main monitor until the problem appeared. Initially when the stash overlay was on the main monitor I could click it (click edit and move it) but once I moved it onto a monitor to the left of the main monitor it stopped being clickable (including the edit button). Interestingly the other overlay with  the icons worked fine on a monitor to the left.
